region/maridia/outer/Mama Turtle Room.json

```diff
@@ -643,30 +592,131 @@
       "name": "Suitless Back on the Back",
       "requires": [
         "canSuitlessMaridia",
-        "canTrickyJump"
+        "canTrickyJump",
```

Comment:
It could be I just don't know the trick here, but trying to get back on Mama Turtle's back seems terrible. Even without any items, it seems much easier to dodge Mama Turtle by jumping over, using the babies as stepping stones (spin jumping off them and breaking spin before landing on the next). Maybe this strat could be removed and "canTrickyJump" could be added as an option to "Suitless Dodge" strat.